tests/system_tests/epics/eiger/test_eiger_system.py

```diff
@@ -65,6 +120,7 @@ async def test_eiger(RE, ioc_prefixes) -> EigerDetector:
     return test_eiger
 
 
+@pytest.mark.skip_as_not_ready
```

Comment:
Can we replace with `@pytest.mark.skip(reason="Not ready, to be implemented in #<issue number>")`?